osquery/filesystem/fileops.h

```diff
@@ -109,8 +109,6 @@ typedef struct {
 /// Constant for an invalid handle.
 const PlatformHandle kInvalidHandle = (PlatformHandle)-1;
 
-std::string lastErrorMessage(unsigned long);
```

Comment:
@directionless, is removing `lastErrorMessage` in this PR intentional?

Reply:
It's not gone, it's moved to the string helpers. I don't see anything using it, and it seems more appropriate elsewhere. 

Did I miss a caller?